sdks/python/examples/evaluate_existing_experiment.py

```diff
@@ -0,0 +1,20 @@
+from opik.evaluation import evaluate_experiment
+from opik.evaluation.metrics import base_metric, score_result
+
+
+class MyCustomMetric(base_metric.BaseMetric):
+    def __init__(self, name: str):
+        self.name = name
+
+    def score(self, **ignored_kwargs):
+        # Add you logic here
+
+        return score_result.ScoreResult(
+            value=10, name=self.name, reason="Optional reason for the score"
+        )
+
+
+evaluate_experiment(
+    experiment_name="surprised_herb_1466",
```

Comment:
[nits] Hi @jacques-comet, I have been using the comet dashboard and it seems like the experiment names can be same even for different runs. Is the intention of this function to reevaluate scores for all matching names? 
<img width="659" alt="Screenshot 2024-11-14 at 15 27 09" src="https://github.com/user-attachments/assets/8dda89ed-0e0e-4ae6-8b64-0e9204b7c199">

Reply:
@JehandadK Yes, this function would re-evaluate all experiments with the same name. I'll do a follow up PR to support experiment IDs (and small FE update to show these ideas) so that you can re-score only one of these experiments